topics/proteomics/tutorials/multiGSEA-tutorial/tutorial.md

```diff
@@ -0,0 +1,138 @@
+>    ```
+{: .hands_on}
+
+
+# Running MultiGSEA
+
+In this step, you'll use the MultiGSEA tool to perform GSEA-based pathway enrichment on the uploaded datasets.
+
+> <hands-on-title> Task description </hands-on-title>
+>
+> 1. Run {% tool [multiGSEA](toolshed.g2.bx.psu.edu/repos/iuc/multigsea/multigsea/1.12.0+galaxy0) %} with the following parameters
+>    - *"Select transcriptomics data"*: `Enabled`
+>        - {% icon param-file %} *"Transcriptomics data"*: `Transcriptomics`
+>        - {% icon param-select %} *"Gene ID format in transcriptomics data"*: `SYMBOL`
+>    - *"Select proteomics data"*: `Enabled`
+>        - {% icon param-file %} *"Proteomics data"*: `Proteomics`
+>        - {% icon param-select %} *"Gene ID format in proteomics data"*: `SYMBOL`
+>    - *"Select metabolomics data"*: `Enabled`
+>        - {% icon param-file %} *"Metabolomics data"*: `Metabolomics`
+>        - {% icon param-select %} *"Metabolite ID format"*: `HMDB`
+>    - *"Supported organisms"*: `Homo sapiens (Human)`.
+>    - *"Pathway databases"*: `Kegg`, `Reactome`
+>    - *"Combine p-values method"*: `Stouffer`
+>    - *"P-value correction method"*: `BH`
+>
+>    > <tip-title>About the parameters</tip-title>
+>    > - **Pathway databases**: Databases often contain their own format in which pathway definitions are provided. So you can select a relevant database. For the tutorial we choose the preset "Kegg" and "Reactome".
+>    > - **Combine p-values method**: Choose a method (here `Stouffer` for balanced weighting). To more comprehensively measure a pathway response, multiGSEA provides different approaches to compute an aggregated p value over multiple omics layers. Because no single approach for aggregating p values performs best under all circumstances, Loughin proposed basic recommendations on which method to use depending on structure and expectation of the problem. If small p values should be emphasized, Fisher’s method should be chosen. In cases where p values should be treated equally, Stouffer’s method is preferable. If large p values should be emphasized, the user should select Edgington’s method. Figure 2 indicates the difference between those three methods.
+>    >   ![P-Value](../../images/p-value.png "P-value methods")
+>    > - **P-value correction method** Type I and type II errors depend on each other and thus reducing type I errors through a p value adjustment will likely increase the chance of making a type II error and an appropriate trade-off has to be made. Choose one of the different methods for controlling false discovery rate: For the tutorial choose `BH` (Benjamini-Hochberg).
+>    {: .tip}
+>
+{: .hands_on}
+
+# Mitochondrial stress activates metabolic pathways of amino acid biosynthesis.
+
+>Mitochondrial stress triggers the activation of amino acid biosynthesis and related metabolic pathways, as highlighted by Quiros et al. (Ref findest du in der Vignette), who identified up-regulation of several amino acid related pathways.
```

Comment:
Quiros et al. needs to be added as a citation.

Reply:
https://training.galaxyproject.org/training-material/topics/contributing/tutorials/create-new-tutorial-content/tutorial.html#citations

Reply:
Quirós, P M, M A Prado, N Zamboni, D D’Amico, R W Williams, D Finley, S P Gygi, and J Auwerx. 2017. “Multi-Omics Analysis Identifies ATF4 as a Key Regulator of the Mitochondrial Stress Response in Mammals.” J Cell Biol 216 (7): 2027–45. https://doi.org/10.1083/jcb.201702058.

Reply:
```
@article{LOUGHIN2004467,
title = {A systematic comparison of methods for combining p-values from independent tests},
journal = {Computational Statistics & Data Analysis},
volume = {47},
number = {3},
pages = {467-485},
year = {2004},
issn = {0167-9473},
doi = {https://doi.org/10.1016/j.csda.2003.11.020},
url = {https://www.sciencedirect.com/science/article/pii/S0167947303002950},
author = {Thomas M. Loughin}
}
```